pyproject.toml

```
@@ -31,6 +31,7 @@ dependencies = [
     "pydantic_settings",
     "argcomplete>=3",
     "typing-extensions; python_version < '3.11'",
+    "esp-bool-parser~=0.1.0"
```

```suggestion
    "esp-bool-parser~=0.1"
```

I don't think we'll have any breaking changes between the minor version releases.

---

Sure